syntax = "proto3";
package orders;
message Order {
  int32 id = 1;
  string product_name = 2;
  int32 qty = 3;
  int32 price = 4;
  int32 total_price = 5;
  int32 user_id = 6;
}
message OrderId  {
  int32 id = 1;
}
message OrderList {
  repeated Order order = 1;
}

message Empty{}

service OrderService{
  rpc add_order(Order) returns (Order);
  rpc delete_order(OrderId) returns (OrderId);
  rpc get_orders(Empty) returns(OrderList);

}
